Camera

linyh0207 opened the pull request:
After the user clicks on the Camera icon from the buttonTabNavigator, it will open up the camera screen. Camera Screen has a button to take a picture and option to flip to the back or front camera. Also, a small overlay message display on the button of the screen. After the picture has been taken, it will change to add post view. It will display the user's current location (hard code) with an edit button. Once the edit button clicked, it will switch to the search bar (text placeholder). There will be discount price and regular price text input, tag entry bar placeholder, and cancel button to trigger the camera view again.

Michael-Xie commented on screens/AddPostScreen.js:
```diff
@@ -0,0 +1,184 @@
+              <View
+                style={{
+                  borderWidth: 2,
+                  borderRadius: '50%',
```
![Screenshot from 2020-06-23 08-08-59](https://user-images.githubusercontent.com/6239834/85402029-fb709480-b528-11ea-9b09-40020c574b98.png)

might be related to this too: https://github.com/facebook/react-native/issues/23219

linyh0207 replied:
It's an issue related to Android. I fixed with break it down the borderRadius: https://github.com/react-native-community/react-native-video/issues/685.

Michael-Xie commented on App.js:
```diff
@@ -26,8 +26,13 @@ export default function App(props) {
         <NavigationContainer>
-          <Stack.Navigator initialRouteName={isAuth ? 'Home' : 'Onboarding'}>
-            <Stack.Screen name="Onboarding" component={OnboardingScreen} />
+          <Stack.Navigator initialRouteName={isAuth ? 'Home' : 'Onboarding'} headerMode="none">
+            {/* Onboarding screen do not need the header */}
```
top menu button is unreachable as a result

linyh0207 replied:
Fix the error by applying SafeAreaView for each screen. https://reactnavigation.org/docs/handling-safe-area/

Michael-Xie commented:
Yay, it works :D
- good that there is a cancel button
- there should be a 'Post' button too
- a minor fix for camera, the live view sees the stretched version and the taken picture returns back to normal

linyh0207 commented:
- Added 'Post' button which reset the Camera Screen view and then navigator back to product main screen. 
- The display image was originally displayed at 75%, it is now fixed to 100%.
